cmake/EthOptions.cmake

```diff
@@ -34,6 +38,10 @@ macro(configure_project)
 		add_definitions(-DETH_VMTRACE)
 	endif ()
 
+	if (JSCONSOLE)
+		add_definitions(-DETH_JSCONSOLE)
```

Comment:
I'm not sure if build without jsconsole is still possible. I also removed `JSCONSOLE` variable from default values.